### ISSUE TITLE
Content disappeared from Tag notes

### ISSUE THREAD

**Issue by Bad3r** [logseq/db-test#131]
### What Happened?

since the updates from yesterday, I noticed content no longer showing up in some tags' Notes section. The content is still saved in the DB and once I navigate to it via search or unlinked references it shows up again on the page. When restarting logseq it disappears again.

see video:

https://github.com/user-attachments/assets/164c3629-fa4c-424b-b8ba-ebc30bc764d3

### Reproduce the Bug

No idea how you can reproduce the bug.

**Comment by Bad3r** [comment]
Update: Exporting and re-importing the graph makes the unlinked references disappear. i.e the data is not getting exported effectively lost.

**Comment by tiensonqin** [comment]
@Bad3r Sorry for the late response, can you try the latest website to confirm whether the bug still exists? Thanks.

**Comment by tiensonqin** [comment]
@Bad3r  I just pushed a fix, try https://logseq-db-demo.pages.dev/#/ in 10 minutes.

**Comment by Bad3r** [comment]
> @Bad3r I just pushed a fix, try [logseq-db-demo.pages.dev#/](https://logseq-db-demo.pages.dev/#/) in 10 minutes.

Thank you! I just tested and the issue seems to be fixed!
some of the content completely disappeared and no longer is visible even in unlinked references. Some of it stayed. Not sure why. I think the content that stayed is content I recently viewed. 

Even if lost content is not recoverable the issue of it disappearing is hopefully solved and wont happen again
